apps/admin/cli/deploy.ts

```diff
@@ -39,6 +39,7 @@ export default {
         await uploadFolderToS3({
             path: buildFolderPath,
             bucket: adminOutput.appStorage,
+            acl: "private",
```

Comment:
Let's make this setting the default one so that the user doesn't have to pass it manually. Ultimately, it's probably better to have a safe default value instead of more convenient one.

Reply:
So, internally, make the default `acl` value to be `private`, and then scan existing `deploy.ts` files and simply remove the `acl` property.

Note that you'll need to do this 4 times in total - 2x in our webiny-js repo, a 2x in `cwp-template-aws` package, which is what gets executed when users run `npx create-webiny-project ...`.

Reply:
I'm not sure we can make it private already, because users need to change their own pulumi code to make private S3 bucket access available. Without proper code migration their apps will stop working. That is why I started with modifying templates only.
When we migrate to simpler pulumi configuration with ready-made components, then I think we can make this change safely.

Reply:
Makes sense. Good catch. 👍🏻